### PR TITLE
fix: update create react app version

### PR DESCRIPTION
**Pull request by stewart-adaptive**
Multiple fixes for new lint warnings and some typescript issues

Changed export default arrow functions to exporting named functions, eg src/client/src/apps/MainRoute/components/reconnect-modal/connect.tsx

Renamed some duplication between types and const, eg src/client/src/apps/MainRoute/data/compositeStatus/actions.ts 

Refactored throttledSendRequest  o satisfy exhaustive-deps lint rule

Removed unused duplicate primary from themes.ts

tsconfig fix for https://github.com/facebook/create-react-app/issues/9429

Disabled unused var issue in openfin.d.ts - Is there a better fix for this???

### CODE REVIEW

File: src/client/package.json [hunk 1]
```diff
@@ -99,12 +100,12 @@
...
-    "@storybook/addon-actions": "^5.2.8",
-    "@storybook/addon-knobs": "^5.2.8",
-    "@storybook/addon-links": "^5.2.8",
-    "@storybook/addon-notes": "^5.2.8",
-    "@storybook/addons": "^5.2.8",
-    "@storybook/react": "^5.3.18",
+    "@storybook/addon-actions": "^6.0.28",
+    "@storybook/addon-knobs": "^6.0.28",
+    "@storybook/addon-links": "^6.0.28",
+    "@storybook/addon-notes": "^5.3.21",
```
Comment (rikoe) [inline src/client/package.json:106]: Is there a reason why this package's version doesn't match the others?

Comment (stewart-adaptive) [reply src/client/package.json:106]: there is no 6.0.28 of that package, 5.3.21 is the most recent

Comment (stewart-adaptive) [reply src/client/package.json:106]: just checked their github
NOTE: addon-notes has been superceded by addon-docs, which fixes numerous bugs, supports a richer feature set, and is easier to maintain. Please give it a try! https://medium.com/storybookjs/rich-docs-with-storybook-mdx-61bc145ae7bc

Comment (stewart-adaptive) [reply src/client/package.json:106]: Ill switch to addon-docs 6.0.28

**Review by rikoe — APPROVED** [review]

**Pull request merged**